fastlane/Fastfile

```diff
@@ -587,6 +600,10 @@ class Build
 
     # Helpers
 
+    def pr_build
```

Comment:
Optional: There is a [convention](https://github.com/rubocop/ruby-style-guide?tab=readme-ov-file#predicate-methods-suffix) in ruby that methods that return a boolean are named with a question mark. E.g.

```ruby
def pr_build?
```

Reply:
Yes, I see. But it looks like we don't much follow it 🤔

Reply:
Yeah it probably doesn't matter that much sticking to conventions considering how much ruby we have. Thanks for making the change anyway!